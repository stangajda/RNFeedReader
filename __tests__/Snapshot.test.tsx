import {it} from '@jest/globals';
import renderer from 'react-test-renderer';

import {Button} from 'react-native';

import React from 'react';

describe('App snapshot test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Button title="hello" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
