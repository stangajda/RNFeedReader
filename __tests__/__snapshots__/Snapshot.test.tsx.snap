// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test renders correctly 1`] = `
<View
  accessibilityRole="button"
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "opacity": 1,
    }
  }
>
  <View
    style={
      [
        {},
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#007AFF",
            "fontSize": 18,
            "margin": 8,
            "textAlign": "center",
          },
        ]
      }
    >
      hello
    </Text>
  </View>
</View>
`;
